nf_core/lint/readme.py

```diff
@@ -17,16 +17,12 @@ def readme(self):
 
            [![Nextflow](https://img.shields.io/badge/nextflow-%E2%89%A50.27.6-brightgreen.svg)](https://www.nextflow.io/)
 
-    * Bioconda badge
+    .. note:: This badge are a markdown image ``![alt-text](<image URL>)`` *inside* a markdown link ``[markdown image](<link URL>)``, so a bit fiddly to write.
 
-      * If your pipeline contains a file called ``environment.yml`` in the root directory, a bioconda badge is required
```

Comment:
Aren't we testing this anymore?

Reply:
nope, that test code is gone (removed since [2.0](https://github.com/nf-core/tools/commit/d7f92158da88dab22c85bce1dc04e9f44a00a5e4))